shared/general/host-ssh.md

```diff
@@ -0,0 +1,15 @@
+### Accessing the host OS
+
+Starting with CLI release 6.12.0, it is possible to SSH into the host OS rather than the application container. This gives you access to logs and tools for services that operate outside the scope of your application, such as **NetworkManager**, **Docker**, and the supervisor.
```

Comment:
we should say "as well as the application container" since they can still SSH into the container as well, but now have the additional option of SSHing into the host